### PR TITLE
Add google groups management example.

### PR DESCRIPTION
**Pull request by LightFelicis**

### CODE REVIEW

File: functions/googleGroupsManager.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+exports.listUsers = async (auth, googleService, domain) => {
+  const service = googleService.admin({version: 'directory_v1', auth});
+  return service.users.list({
+    domain: domain,
+    customer: 'my_customer',
+    orderBy: 'email',
+  }).then(
+      (response) => {
+        return response.data.users;
+      },
```
Comment (leon-kastler-google) [inline functions/googleGroupsManager.js:18]: Could be like this, no?
```suggestion
      (response) => response.data.users,
```

File: functions/googleGroupsManager.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+/**
+ function that retrieves all users from the domain's gSuite
+ using Admin Directory SDK
+ * @param {OAuth2.0 instance} auth
+ * @param {gAPI instance} googleService
+ * @param {string} domain
+ * @return {Promise<any>} the found list of users or null otherwise
+ */
+exports.listUsers = async (auth, googleService, domain) => {
+  const service = googleService.admin({version: 'directory_v1', auth});
```
Comment (leon-kastler-google) [inline functions/googleGroupsManager.js:10]: `'directory_v1'` should be a constant, defined outside of this function

File: functions/index.js [hunk 3]
```diff
@@ -15,22 +20,53 @@ const SCOPES = [
...
+function checkCredentials(path, callback) {
   fs.readFile('credentials.json', (err, content) => {
     if (err) {
       return console.log('Error loading client secret file:', err);
     }
-    authorize(JSON.parse(content), (auth) => {
-      sendMessage(auth, recipient);
+    return authorize(JSON.parse(content), (auth) => {
+      return callback(auth);
     });
```
Comment (leon-kastler-google) [inline functions/index.js:67]:
```suggestion
    return authorize(JSON.parse(content), (auth) => callback(auth));
```

**Review by leon-kastler-google — APPROVED** [review]
Looks good!

File: functions/googleGroupsManager.js (hunk 2 above)
Comment (LightFelicis) [reply functions/googleGroupsManager.js]: Done.

**Pull request merged**